CHANGELOG.md

```diff
@@ -1,16 +1,18 @@
 # Changelog
 
-## Unreleased
+## 3.0.0 (Breaking release)
 
 💥 Breaking changes:
 
-- Pull Request Title goes here
+- Rename $govuk-border-width-mobile to $govuk-border-width-narrow
 
-  Description goes here (optional)
+  This better reflects how the variable is used.
 
-  To migrate you need to change: X
+  This PR also makes the error summary border the standard width on mobile.
```

Comment:
Non-blocking in future could split the 'fix' out into a separate changelog entry

Reply:
Thanks @nickcolley I've split this into breaking change and fix 👍